deployment/gcloud-deploy.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Deploying with Google Cloud
+---
+
+Deploying your Authmaker app is simple with [Google Cloud Platform](https://cloud.google.com/). With Google Cloud, hosting for small web applications is free (in most cases) - you won't have to pay until your application scales past their [free tier](https://cloud.google.com/free/).
+
+If you don't have an existing Google Cloud account, you will need to create one before the next steps.
+
+## Configure your DNS
+
+You will need to own a domain name to use as the URL for your deployed application. You can register a new domain with [Google Domains](https://domains.google.com) if you do not already have one, but any DNS provider you choose is fine.
+
+If you are using a DNS provider other than Google, you will need to verify ownership of the domain through [Google Webmaster Central](https://www.google.com/webmasters/verification/). If you created your domain with Google, this step is automatic.
+
+#### Create a CNAME record
```

Comment:
So we have a bit of an issue with this segment... and I don't know how to address it 🤔 

we probably need to add the ability to have like a ⚠️ WARNING section to the guides or something like that.

Essentially the problem is that you can't have an **apex** cname. I can't remember if we have discussed this before 🤔 there are some workarounds out there but _none of them work on gcloud_ 😭 

This is probably something to add to the discussion pile